src/org/labkey/test/components/ui/files/FileUploadPanel.java

```diff
@@ -78,6 +78,16 @@ public File downloadTemplate()
         return getWrapper().doAndWaitForDownload(()->elementCache().downloadTemplate.click());
     }
 
+    /*
```

Comment:
adding this method was the least-intrusive way I could make the framework able to work with the existence of a second attachment container

Reply:
If this is too much, I could remove this method here and add it to the page class in Bio